core/binary.mk

```diff
@@ -38,6 +38,7 @@ my_static_libraries := $(LOCAL_STATIC_LIBRARIES)
 my_whole_static_libraries := $(LOCAL_WHOLE_STATIC_LIBRARIES)
 my_shared_libraries := $(LOCAL_SHARED_LIBRARIES)
 my_cflags := $(LOCAL_CFLAGS)
+my_cflags := -w $(filter-out -Wall -Werror -Wextra -Weverything,$(my_cflags))
```

Comment:
https://github.com/BenzoRom/build/commit/9331a353405d1fc204fd25c20f8f2fd5fcf28c3b
Found the original owner of the commt